Directory.Build.props

```diff
@@ -2,22 +2,30 @@
 <Project>
 
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <RepositoryUrl>https://github.com/dotnet/sdk</RepositoryUrl>
-    <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
-
     <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
     <LangVersion>Latest</LangVersion>
   </PropertyGroup>
 
-  <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
     <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+
+    <!--
+      Tools and packages produced by this repository support infrastructure and are not shipping on NuGet or via any other official channel.
+    -->
+    <IsShipping>true</IsShipping>
```

Comment:
I don't understand this comment vs. IsShipping=true.

Reply:
Yes, on the other thread, we agreed to keep IsShipping to true and to come up/set another property to indicate that the packages produced here are not going to NuGet, which is why the comment is there. 

Full disclosure, the props files, I took from @chcosta who started on the process of moving the SDK to arcade. But I think this comment is still relevant.

Reply:
I agree with IsShipping=true, but the comment seems to suggest the opposite. This repository ddoes more than support infrastructure.

Reply:
Sounds good. I will replace the comment with an issue to figure out this separate property to determine the nupkgs from here shuold not go to nuget.org. Sounds good?